spec/models/court_spec.rb

```diff
@@ -87,7 +99,12 @@
         end
 
         context 'the API returned emails' do
-          let(:api_response) { {'emails' => [{"description" => "applications", "address" => "applications@email"}]} }
+          let(:api_response) do
+            {
+              'emails' => [{"description" => "applications", "address" => "applications@email"}],
+              'gbs' => nil
```

Comment:
I think no need to add the gbs code here, as this is testing only the email response.